app/code/Magento/Catalog/Block/Product/ListProduct.php

```diff
@@ -7,13 +7,17 @@
 namespace Magento\Catalog\Block\Product;
 
 use Magento\Catalog\Api\CategoryRepositoryInterface;
+use Magento\Catalog\Block\Product\Context;
```

Comment:
Import of Magento\Catalog\Block\Product\Context is not necessary.

Reply:
Solved. Thank you for your review.